src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div class="ec-checkbox">
+    <input
+      :id="id"
+      v-model="inputModel"
+      :aria-describedby="errorId"
+      type="checkbox"
+      class="ec-checkbox__input"
+      :disabled="disabled"
+    >
+    <label
+      :for="id"
+      class="ec-checkbox__label"
+      :class="{
+        'ec-checkbox__label--has-error': isInvalid || $slots['error-message'],
+        'ec-checkbox__label--large': large
```

Comment:
I don't see the need of using a `large` prop, all checkboxes should behave the same, and actually this can be detrimental as the length may vary between languages.